server/sonar-main/src/main/java/org/sonar/application/process/ProcessLauncherImpl.java

```diff
@@ -104,40 +127,30 @@ private static void cleanupOutdatedEsData(EsCommand esCommand) {
     });
   }
 
-  private static void writeConfFiles(EsCommand esCommand) {
-    EsFileSystem esFileSystem = esCommand.getFileSystem();
-    File confDir = esFileSystem.getConfDirectory();
+  private static void writeConfFiles(ElasticsearchConfiguration elasticsearchConfiguration) {
+    File confDir = elasticsearchConfiguration.getConfDirectory();
     if (!confDir.exists() && !confDir.mkdirs()) {
       String error = format("Failed to create temporary configuration directory [%s]", confDir.getAbsolutePath());
       LOG.error(error);
       throw new IllegalStateException(error);
     }
 
     try {
-      esCommand.getEsYmlSettings().writeToYmlSettingsFile(esFileSystem.getElasticsearchYml());
-      esCommand.getEsJvmOptions().writeToJvmOptionFile(esFileSystem.getJvmOptions());
-      esCommand.getLog4j2Properties().store(new FileOutputStream(esFileSystem.getLog4j2Properties()), "log4j2 properties file for ES bundled in SonarQube");
+      elasticsearchConfiguration.getEsYmlSettings().writeToYmlSettingsFile(elasticsearchConfiguration.getElasticsearchYml());
+      elasticsearchConfiguration.getEsJvmOptions().writeToJvmOptionFile(elasticsearchConfiguration.getJvmOptions());
+      elasticsearchConfiguration.getLog4j2Properties().store(new FileOutputStream(elasticsearchConfiguration.getLog4j2PropertiesLocation()), "log4j2 properties file for ES bundled in SonarQube");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 195 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)